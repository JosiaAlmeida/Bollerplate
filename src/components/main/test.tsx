import {render, screen} from '@testing-library/react'

import Main from '.'

//Component what should be test
describe('<Main />', ()=>{
    //Partial in test
    it('should render the heading',()=>{
        render(<Main />)
        expect(
            screen.getByRole('heading',{name: /react avancado/i})
            
        )
    })
    it('should make a snapShot', ()=>{
        const {container} = render(<Main />)
        expect(container.firstChild).toMatchSnapshot()
    })
})