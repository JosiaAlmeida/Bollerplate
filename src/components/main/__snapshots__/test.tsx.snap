// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should make a snapShot 1`] = `
<main>
   
  <h1>
    react avancado
  </h1>
   
</main>
`;
